arbnode/inbox_test.go

```diff
@@ -138,12 +138,12 @@ func NewTransactionStreamerForTest(t *testing.T, ctx context.Context, ownerAddre
 		},
 	}
 
-	chainDb := rawdb.NewMemoryDatabase()
-	arbDb := rawdb.NewMemoryDatabase()
+	executionDB := rawdb.NewMemoryDatabase()
+	consensusDb := rawdb.NewMemoryDatabase()
```

Comment:
Same here, consensusDB instead of consensusDb.

Reply:
And done for `melDb` as well